frontend/src/queries/nodes/HogQLQuery/HogQLQueryEditor.tsx

```diff
@@ -44,7 +44,8 @@ export function HogQLQueryEditor(props: HogQLQueryEditorProps): JSX.Element {
         <div className="space-y-2">
             <div
                 data-attr="hogql-query-editor"
-                className={'flex flex-col p-2 border rounded bg-bg-light space-y-2 resize-y w-full'}
+                className={'flex flex-col p-2 border rounded bg-bg-light space-y-2 resize-y w-full overflow-hidden'}
+                style={{ height: 318 }}
```

Comment:
Why this height in particular?

Reply:
Oh... I didn't have the AI bar enabled. It does fit the default query if that's not there

Reply:
Makes sense, since it's on just for us now